packages/code-gen/package.json

```diff
@@ -1,7 +1,16 @@
 {
-  "name": "code-gen",
-  "version": "0.0.1",
+  "name": "@dynodm/code-gen",
+  "version": "0.0.0",
+  "description": "code generator -- converts models into workable typescript",
+  "main": "index.js",
+  "repository": "https://github.com/Trunkrs/DynoDM/tree/master/packages/code-gen",
+  "license": "MIT",
+  "scripts": {
+    "tsc": "tsc"
+  },
   "dependencies": {
-    
+    "typescript": "^3.9.6"
```

Comment:
Maybe you could remove this dependency? 🙂 As the root has it already defined under `devDependencies` 🙂

Reply:
No, the package will need it as the code generation is done through the typescript API